Components/badges.css

```diff
@@ -0,0 +1,92 @@
+.badge-container {
+  position: relative;
+  cursor: pointer;
+}
+
+.badge-red {
+  background-color: var(--dark-danger-color);
+}
+.badge-green {
+  background-color: var(--success-color);
+}
+.badge-yellow {
+  background-color: var(--warning-color);
+}
+.badge-pink {
+  background-color: var(--primary-color);
+}
+
+/* Badges on Avatar */
+
+.badge-avatar {
+  border-radius: 50%;
+  padding: 2px;
+  border: 2px solid white;
+  position: absolute;
+}
+
+.badge-container .lg-avatar ~ .badge-avatar {
+  height: 1.5rem;
+  width: 1.5rem;
+  bottom: 0.9em;
```

Comment:
If you are using rem unit below 1rem then try to use px unit

Reply:
ok i will take care of